pontoon/base/static/js/translate.js

```diff
@@ -3953,6 +3955,17 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Update title of the current view.
+     */
+    updateTitle: function() {
```

Comment:
@mathjazz Do you wish to keep that function or should I move it into  process entities?

Reply:
I'd keep it. The PR looks OK to me.